compiler+runtime/bin/jank/compiler+runtime/bash_test.clj

```diff
@@ -25,19 +26,25 @@
           (if skip?
             (util/log-warning "Skipped " relative-dirname)
             (util/with-elapsed-time duration
-              (let [res @(b.p/process {:out :string
-                                       :err :out
-                                       :dir dirname
-                                       :extra-env extra-env}
-                                      test-file)]
-                (when (or (and (zero? (:exit res)) expect-failure?)
```

Comment:
I don't understand why we need timeouts for this. None of your exit code stuff is blocking, looping, etc. What's the use case for being concerned about it not returning fast enough?

Reply:
jank's CI *does not* fail fast. We specifically have this:

```yml
jobs:
  test:
    strategy:
      fail-fast: false
```

Reply:
Yes, sorry. I was looking at the wrong build where I changing the `timeout-minutes` to be lower.

Here's a better example: https://github.com/jank-lang/jank/actions/runs/13209300118/job/36879582648#step:3:46

The raw logs show we were for 25m while testing `main-exit`. I still have no idea why it happened. This was the motivation for the timeout changes. Mysteriously the timeouts stopped around when I added the `apt-get update`.

```
2025-02-07T22:52:39.9143327Z 🛈  Tested module/cljc (542ms)
2025-02-07T23:16:25.4280703Z ##[error]The operation was canceled.
```

Reply:
I suspect it was waiting for user input. Maybe because it was trying to install something?

Reply:
Yes. The timeout changes were to prevent silly mistakes like this from melting CI machines, which might be better handled higher up in the script. Leaving as-is for now.

Reply:
Our CI workflow file has this:

```yml
    timeout-minutes: 30
```

Analysis builds take ~25m, while everything other build finishes in ~7m, and linting takes ~3m (most of which is spent installing apt deps). We could perhaps configure this per-job, like we do the name, so that it's 30 for analysis builds and 10 for normal builds and 5 for linting.